changelog/fragments/bundle-validate-optional-validators.yaml

```diff
@@ -0,0 +1,11 @@
+entries:
+  - description: >
+      Added the `--select-optional` flag to `bundle validate` to pass a label selector
+      that runs any matching optional validator on top of required validators,
+      and `--list-optional` to display optional validators, their selectors, and
+      a short explanation of what each does.
+    kind: addition
+  - description: >
+      Added the OperatorHub.io optional validator to `bundle validate`, which can be run
+      by setting `--select-optional=name=operatorhub` or `--select-optional=suite=operatorframework`.
+    kind: addition
```

Comment:
HI @estroz, 

Would be nice add the checks for this feature in the e2e tests for all types since it would ensure the quality of the provided features with Helm/Ansible and GO. Could we do that?